core/library_steam.gd

```diff
@@ -459,3 +467,25 @@ func _app_supports_linux(app_id: String) -> bool:
 		return false
 	
 	return info[app_id]["data"]["platform"]["linux"]
+
+
+## Hook to execute before app launch
```

Comment:
Please add more information, such as the reason it is needed and what it's doing

Reply:
Added. Right now this app lifecycle hook will just try to make sure the app is up-to-date before launching it. In the future, we can also add shader compilation, etc.